content/en/docs/reference/glossary/container-runtime.md

```diff
@@ -15,7 +15,7 @@ tags:
 
 <!--more-->
 
-Kubernetes supports several container runtimes: {{< glossary_tooltip term_id="docker">}},
+Kubernetes supports several container runtimes: {{< glossary_tooltip term_id="docker">}} ([Deprecated](https://kubernetes.io/blog/2020/12/02/dockershim-faq/)),
```

Comment:
Is Docker deprecated? AIUI, using Docker continues to work as your container runtime so long as you use `dockershim`, which [is now maintained by Mirantis](https://www.mirantis.com/blog/mirantis-to-take-over-support-of-kubernetes-dockershim-2/).

The change is about where you get `dockershim` from, because the one built in to Kubernetes is now deprecated.

Reply:
Oh okay. Let me close this PR then.